src/werkzeug/middleware/lint.py

```diff
@@ -117,7 +117,7 @@ def close(self) -> None:
 
 
 class GuardedWrite:
-    def __init__(self, write: t.Callable[[bytes], None], chunks: t.List[int]) -> None:
+    def __init__(self, write: t.Callable[[bytes], object], chunks: t.List[int]) -> None:
```

Comment:
I'm just curious, how about using `Any` here?

Reply:
That would work too, but object is preferable. `Any` basically means "I can't represent the true type in the current type system", while `object` tells the type checker "only let me do things with this object that are valid for every single object".